Makefile

```diff
@@ -28,18 +30,18 @@ VERSION := $(shell date '+%Y.%m.%d.%H')
 
 check: shellcheck shfmtcheck
 
-package:
+packages: $(addprefix package-,$(ALL_PLATFORMS))
```

Comment:
We'll also want to update `linux-pkg` to call `make packages` instead of `make package`

Reply:
👍 https://github.com/delphix/linux-pkg/pull/4